webtau-graphql/src/main/java/org/testingisdocumenting/webtau/graphql/GraphQLReportDataProvider.java

```diff
@@ -28,27 +28,28 @@
 import java.util.LongSummaryStatistics;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Supplier;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 public class GraphQLReportDataProvider implements ReportDataProvider {
-    private final GraphQLCoverage coverage;
+    private final Supplier<GraphQLCoverage> coverage;
 
     public GraphQLReportDataProvider() {
-        this(GraphQL.getCoverage());
+        this(GraphQL::getCoverage);
```

Comment:
This is what fixed the local mvn run problem.  I can only assume in Travis we somehow fork a jvm per test class.  It's the only explanation I have for this problem not happening there.